README.md

```diff
@@ -18,6 +18,12 @@ Running on the *host* interface (recommended):<br/>
 $ docker run -d --net=host --name=squad-dedicated cm2network/squad
 ```
 
+Running as a normal docker container:<br/>
+
+```console
+$ docker run -d -p 7787:7787/udp -p 7788:7788/udp -p 27165:27165/tcp -p 27165:27165/udp -p 27166:27166/tcp -p 27166:27166/udp -p 21114:21114/tcp -p 21114:21114/udp -e PORT=7787 -e QUERYPORT=27165 -e RCONPORT=21115 --name=squad-server cm2network/squad
```

Comment:
Same comment as below, what's the reason for you opening port 7788 and 27166? "QueryPort+1" Is only required if you run multiple server instances...

Reply:
I expose all ports like described in the wiki: https://squad.gamepedia.com/Server_Configuration#Ports_to_Open
Also Port 7788 and 27166 are used if there are problems with the lower default port.

Reply:
See my reply above